test/JDBC/input/bbf_view_def.mix

```diff
@@ -0,0 +1,168 @@
+-- tsql
+create table vd_t1 (c1 int primary key);
```

Comment:
Please drop this table at the end. It leads to one failure in JDBC.

Reply:
Done.